Packages/com.unity.inputsystem/InputSystem/Actions/Interactions/HoldInteraction.cs

```diff
@@ -82,7 +82,7 @@ public void Process(ref InputInteractionContext context)
                     {
                         context.PerformedAndStayPerformed();
                     }
-                    else if (!context.ControlIsActuated())
+                    if (!context.ControlIsActuated())
                     {
                         // Control is no longer actuated and we haven't performed a hold yet,
```

Comment:
This comment isn't really accurate anymore because we might have just now performed the hold

Reply:
Good catch. Fixed.